platform/app/models/connector.rb

```
@@ -44,4 +44,4 @@ def ensure_default_analytics_db_is_false_for_api_type
 
     self.default_analytics_db = false
   end
-end
+end
```

rubocop

[convention] Final newline missing.

(Layout/TrailingEmptyLines)